docs/01/b.md

```diff
@@ -2,13 +2,13 @@
   [algebra]: https://github.com/non/algebra
   [Equivalence]: http://en.wikipedia.org/wiki/Equivalence_relation
 
-## Eq
+## `Eq`
 
 LYAHFGG:
 
 > `Eq` is used for types that support equality testing. The functions its members implement are `==` and `/=`.
 
-Cats equivalent for the `Eq` typeclass is also called `Eq`.
+Cats' equivalent for the `Eq` typeclass is also called `Eq`.
```

Comment:
https://owl.english.purdue.edu/owl/resource/621/01/

`add 's to the singular form of the word (even if it ends in -s):
the owner's car 
James's hat (James' hat is also acceptable. For plural, proper nouns that are possessive, use an apostrophe after the 's': "The Eggleses' presentation was good." The Eggleses are a husband and wife consultant team.)`

If we assume that "Cats" is singular, we could just go with `Cats's`